src/test/regress/sql/pg17.sql

```diff
@@ -499,81 +499,6 @@ RESET client_min_messages;
-    id serial NOT NULL,
-    partition_col int NOT NULL,
-    PRIMARY KEY (id, partition_col)
-) PARTITION BY RANGE (partition_col);
--- Add partitions to the local partitioned table
-CREATE TABLE local_partitioned_table_p1 PARTITION OF local_partitioned_table
-FOR VALUES FROM (1) TO (100);
-CREATE TABLE local_partitioned_table_p2 PARTITION OF local_partitioned_table
-FOR VALUES FROM (100) TO (200);
-SELECT citus_add_local_table_to_metadata('local_partitioned_table');
-
--- Verify the Citus tables
-SELECT table_name, citus_table_type FROM pg_catalog.citus_tables
-WHERE table_name::regclass::text LIKE '%_partitioned_table' ORDER BY 1;
-
--- Step 3: Add an exclusion constraint with a name to the distributed partitioned table
-ALTER TABLE distributed_partitioned_table ADD CONSTRAINT dist_exclude_named EXCLUDE USING btree (id WITH =, partition_col WITH =);
-
--- Step 4: Verify propagation of exclusion constraint to worker nodes
-\c - - :public_worker_1_host :worker_1_port
-SET search_path TO pg17;
-SELECT conname FROM pg_constraint WHERE conrelid = 'pg17.distributed_partitioned_table'::regclass AND conname = 'dist_exclude_named';
-
--- Step 5: Add an exclusion constraint with a name to the Citus local partitioned table
-\c - - :master_host :master_port
-SET search_path TO pg17;
-ALTER TABLE local_partitioned_table ADD CONSTRAINT local_exclude_named EXCLUDE USING btree (partition_col WITH =);
-
--- Step 6: Verify the exclusion constraint on the local partitioned table
-SELECT conname, contype FROM pg_constraint WHERE conname = 'local_exclude_named' AND contype = 'x';
-
--- Step 7: Add exclusion constraints without names to both tables
-ALTER TABLE distributed_partitioned_table ADD EXCLUDE USING btree (id WITH =, partition_col WITH =);
-ALTER TABLE local_partitioned_table ADD EXCLUDE USING btree (partition_col WITH =);
-
--- Step 8: Verify the unnamed exclusion constraints were added
-SELECT conname, contype FROM pg_constraint WHERE conrelid = 'local_partitioned_table'::regclass AND contype = 'x';
-\c - - :public_worker_1_host :worker_1_port
-SET search_path TO pg17;
-SELECT conname, contype FROM pg_constraint WHERE conrelid = 'pg17.distributed_partitioned_table'::regclass AND contype = 'x';
-
--- Step 9: Drop the exclusion constraints from both tables
-\c - - :master_host :master_port
-SET search_path TO pg17;
-ALTER TABLE distributed_partitioned_table DROP CONSTRAINT dist_exclude_named;
-ALTER TABLE local_partitioned_table DROP CONSTRAINT local_exclude_named;
-
--- Step 10: Verify the constraints were dropped
-SELECT * FROM pg_constraint WHERE conname = 'dist_exclude_named' AND contype = 'x';
-SELECT * FROM pg_constraint WHERE conname = 'local_exclude_named' AND contype = 'x';
-
--- Step 11: Clean up - Drop the tables
-DROP TABLE distributed_partitioned_table CASCADE;
-DROP TABLE local_partitioned_table CASCADE;
--- End of Test for exclusion constraints on partitioned and distributed partitioned tables in Citus environment
-
```

Comment:
I guess this was removed by mistake, right?

Reply:
Yes, will look into that.